python/ray/experimental/gpu_object_manager/gpu_object_store.py

```diff
@@ -110,4 +110,5 @@ def add_gpu_object(self, obj_id: str, gpu_object: List["torch.Tensor"]):
         self.gpu_object_store[obj_id] = gpu_object
 
     def remove_gpu_object(self, obj_id: str):
-        del self.gpu_object_store[obj_id]
+        if obj_id in self.gpu_object_store:
+            del self.gpu_object_store[obj_id]
```

Comment:
worth leaving a comment here (or maybe better at the callsite?) for the known case where the ID is not present